components/hab/src/default_values.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2017-2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This file contains the string defaults values as well as environment variable strings
+//! for use in the clap layer of the application. This is not the final form for defaults.
+//! Eventually this will likely reside in it's own crate and be composed of fully typed
+//! default values. But as a first step we need a spot to consolidate those values and help simplify
+//! some of the logic around them.
+
+// Supervisor
```

Comment:
I don't think this or the `// Supervisor ENV vars` comments add much. If we get to a point where there are tons of things here and we want to break them into sections, it may be worth reconsidering, but right now I see them as not helpful, and I avoiding unhelpful comments leaves more room for seeing code.